test/rql_test/connections/r_http.py.httpbin

```diff
@@ -137,8 +137,8 @@ class TestHttpTerm(rdb_unittest.RdbTestCase):
     def test_redirects(self):
         url = self.getHttpBinURL('redirect', '2')
         
-        self.assertRaisesRegex(r.ReqlRuntimeError, self.err_string('GET', url, 'status code 302'), r.http(url, redirects=0).run, self.conn)
-        self.assertRaisesRegex(r.ReqlRuntimeError, self.err_string('GET', url, 'Number of redirects hit maximum amount'), r.http(url, redirects=1).run, self.conn)
```

Comment:
These APIs were saying they were undefined/missing so updating them to the newer interface